backend/internal/storage/postgres/works.go

```diff
@@ -139,6 +139,15 @@ WHERE student_work_id = $3
 	return formatted[0], nil
 }
 
+// func (db *DB) GetWorkFromRepoName(ctx context.Context, repoName string) (*models.StudentWork, error){
+// 	var studentWork models.StudentWork
+// 	err := db.connPool.QueryRow(ctx, `SELECT * FROM student_works WHERE repo_name = $1`, repoName).Scan(&studentWork)
```

Comment:
Leaving-> This is functional but throws an error because it's unused. No reason to delete working code